CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # v2.2.2 # UNRELEASED CHANGES
 ### Version highlights:
 1. Provider names are now case insensitive
+2. Added the 'oneview' feature
```

Comment:
It seems you implemented a new feature to the integration. It would be better to maybe call Puppet feature or something like that.